docs/shared/packages/gradle/gradle-plugin.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Overview of the Nx Gradle Plugin
+description: The Nx Gradle plugin contains generators and plugins that are useful for Gradle projects in an Nx workspace.
+---
+
+The Nx Gradle plugin contains generators and plugins that are useful for Gradle projects in an Nx workspace.
```

Comment:
Please add a quick description of Gradle. Most other plugin overview pages also give a quick description of the underlying tool (Gradle).

I also think the main highlight at the moment is the plugin which links gradle tasks to Nx tasks, not generators.

https://nx-dev-git-fork-xiongemi-feature-nxc-544-add-gradle-ce7275-nrwl.vercel.app/nx-api/jest/documents/overview